src/routes/lexicon/+page.svelte

```diff
@@ -271,30 +294,36 @@
         />
     {/if}
 
-    <div
-        class="flex-1 overflow-y-auto bg-base-100"
-        bind:this={scrollContainer}
-        on:scroll={checkIfScrolledToBottom}
-    >
-        {#if selectedWord}
-            <WordNavigationStrip
-                currentWord={selectedWord}
-                wordsList={selectedLanguage === vernacularLanguage
-                    ? vernacularWordsList
-                    : reversalWordsList}
-                onSelectWord={selectWord}
-            />
-            <LexiconEntryView
-                {selectedWord}
-                {vernacularWordsList}
-                {vernacularLanguage}
-                onSwitchLanguage={switchLanguage}
-                onSelectWord={selectWord}
-            />
-        {:else if selectedLanguage === vernacularLanguage}
+    {#if selectedWord}
+        {#if !showBackButton}
+            {(showBackButton = true)}
+        {/if}
```

Comment:
I am not quite sure what you are trying to achieve with showBackButton, but this is not the way to do it.